core/new-gui/src/app/workspace/component/visualization-panel/visualization-panel.component.ts

```diff
@@ -53,6 +53,7 @@ export class VisualizationPanelComponent implements OnChanges {
     this.modalRef = this.modalService.create({
       nzTitle: 'Visualization',
       nzWidth: 1100,
+      nzFooter: null,
```

Comment:
please add a short comment to explain why we need to expliclity set it to null, 
I check the documents and found this is needed if we do not want the footer to show.  But without comment this might be misunderstood as useless code.